hdl/XilinxBuildScript.tmpl

```diff
@@ -35,15 +59,36 @@ report_ip_status
 {{ end }}
 
 {{ range .Constrs }}
-read_xdc "{{ . }}"
+	read_xdc {{ if $outOfContext }} -mode out_of_context {{ end }} "{{ . }}"
 {{ end }}
 
-synth_design -top {{ .Name }} -include_dirs {{ .IncDir }}
-opt_design
-place_design
-phys_opt_design
-route_design
-report_timing_summary -file {{ .Timing }}
-write_bitstream -force bitstream.bit
-write_debug_probes -force bitstream.ltx
+synth_design -top {{ .Name }} -include_dirs {{ .IncDir }} {{ if .OutOfContext }} -mode out_of_context {{ end }} -flatten_hierarchy rebuilt > $WORK/synth.log
+exec mkdir -p $WORK/post_synth
```

Comment:
I'd add all these or at least the directory to the list of dbt outputs.

Reply:
On second thought, I'll probably add the last build step for OOC synthesis too.